src/components/create.vue

```diff
@@ -243,55 +243,54 @@ export default {
         this.$refs.errorModal.show();
         return;
       }
+
+      let blockchain = getBlockchain(this.selectedChain);
+
       try {
-        apkey = PrivateKey.fromWif(this.activepk)
-          .toPublicKey()
-          .toString(this.selectedChain);
-        mpkey = PrivateKey.fromWif(this.memopk)
-          .toPublicKey()
-          .toString(this.selectedChain);
+        apkey = blockchain.getPublicKey(this.activepk);
+        mpkey = blockchain.getPublicKey(this.memopk);
         if (this.includeOwner == 1) {
-          opkey = PrivateKey.fromWif(this.ownerpk)
-            .toPublicKey()
-            .toString(this.selectedChain);
+          opkey =blockchain.getPublicKey(this.ownerpk);
         }
       } catch (e) {
+        console.error(e);
```

Comment:
Need to run this to see if it breaks anything but I reckon we should probably remove this and throw at the end of the catch block instead